src/dso.cc

```diff
@@ -59,7 +61,10 @@ Dso::Dso(pid_t pid, ElfAddress_t start, ElfAddress_t end, ElfAddress_t pgoff,
              // ends with .jsa
              ((_filename.length() > s_jsa_str.length() + 1) &&
               _filename.substr(_filename.length() - s_jsa_str.length(),
-                               _filename.length()) == s_jsa_str)) {
```

Comment:
Why not create a new DSO type for these files ?

Reply:
I guess it is slightly confusing.
I wanted to make sure that any type of symbols managed by runtimes could be overridden. 
The cleaner way could be to have 2 types, and allow both to default to the perf-map symbols.